### PR TITLE
Oracle driver documentation :page_with_curl: [ci skip]

### PR DESCRIPTION
**Pull request by camsaul**
Implements #3269.

@kdoh or @mazameli if one of you guys could review/proofread/&c. really quick and let me know if this works that would be 👍

### CODE REVIEW

File: docs/administration-guide/databases/oracle.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Working with Oracle in Metabase
+
+Starting in v0.20.0, Metabase provides a driver for connecting to Oracle databases. Under the hood, Metabase uses to Oracle's JDBC driver; due to licensing restrictions, we can't
```
Comment (mazameli) [inline docs/administration-guide/databases/oracle.md]: > Metabase uses ~~to~~ Oracle's JDBC driver

File: docs/administration-guide/databases/oracle.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+include ship it as part of Metabase. Luckily, downloading it yourself and making it available to Metabase is straightforward and only takes a few minutes.
```
Comment (mazameli) [inline docs/administration-guide/databases/oracle.md]: > include ~~ship~~ it as part of Metabase.

**Comment by mazameli** [comment]
Besides those line notes, this looks good. 👍

**Comment by camsaul** [comment]
Thanks, I'll fix those 🔧

**Pull request merged**